src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterLocationService.cs

```diff
@@ -121,7 +131,12 @@ public class ImporterLocationService : BaseImporterService
             return string.Join(separator, observationalUnits
                 .Where(unit => unit != null)
                 .Select(unit =>
-                    $"{unit.GetType()}:{(unit is LocalAuthority la ? la.GetCodeOrOldCodeIfEmpty() : unit.Code)}:{unit.Name}"));
+                {
+                    var code = unit.Code;
+                    if (unit is LocalAuthority la) code = la.GetCodeOrOldCodeIfEmpty();
+                    else if (unit is School sch) code = sch.GetCodeOrOldCodeIfEmpty();
```

Comment:
I don't think code which is the urn is ever going to be null, so I don't think laestab will ever be used and we could just not import it/store it.